### DISCUSSION TITLE
Inclusivité du parcours choix du type de restaurant

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1393] [Autres sujets]
J'ai échange avec une utilisatrice ce matin qui m'a fait remarquer qu'il n'y avait pas le cas des cantines qui ne prépare pas ses repas sur place et qui n'est pas non plus servie par une cuisine centrale (le cas des CRC).
Est-ce qu'on pourra rajouter un petit complément ?
"une cantine qui sert des repas préparés par une cuisine centrale ou une société de restauration collective" ?
C'est pas le cas prévu à l'origine mais quand l'utilisatrice s'est enregistré sa première remarque c'était "bah y a déjà un sujet car mon cas n'est pas proposé". 
<img width="741" alt="Capture d’écran 2022-04-06 à 10 50 28" src="https://user-images.githubusercontent.com/85872944/162035574-df2bb334-27cb-44de-9290-b703fb9b0f1a.png">

**Comment by alemangui** [comment]
Je transforme ça en discussion @gauthierandre, car en effet pas sur de quoi faire

**Comment by alemangui** [comment]
@valeriemerle @JenniferStephan avez-vous une idée de quoi faire dans ce cas ? 

On peut reformuler la deuxième option : _"Une cantine qui sert des repas preparés aillers (par ex. dans une cuisine centrale)"_

Ou en ajouter une nouvelle ?

**Reply by JenniferStephan** [reply]
c'est une bonne solution ! mais en soit, une cuisine livrée par une SRC est livrée par une cuisine centrale de la SRC...

**Reply by valeriemerle** [reply]
oui une cantine qui est livrée par une SRC est en fait livrée par la cuisine centrale de la SRC mais elle ne le sait pas forcement. Modif à apporter si bcq de retour mais il semblerait que oui vu les mails retours que l'on a des communes qui ecrivent qu'elles ne se sentent pas concernées par l'inscription dans ma cantine si elles se font livrer les repas!
